src/classes/AccountsController.cls

```diff
@@ -0,0 +1,13 @@
+public class AccountsController {
+  @AuraEnabled
+  public static List<Account> getAccounts(String filter) {      
+      List<Account> accs = new List<Account>();
+      if(!String.isEmpty(filter)) {
+          String newSearchText = '%'+filter+'%';
```

Comment:
why is the "new" word there in that name ? isn't this just a search text ? what makes it new ?